### DISCUSSION TITLE
Please change the License for VREngine

### DISCUSSION THREAD

**Discussion by HeadClot** [Jordonbc/VREngine#7] [General]
Hey @Jordonbc, 
First off thank you for making VREngine, I have been looking for something like this for a while now. 

I need to bring to your attention that GPL licensed code cannot be used with Unreal Engine 4. The following licenses are compatible with UE4: BSD License, MIT License, Microsoft Public License, or Apache License. 

"You may not combine, Distribute, or otherwise use the Licensed Technology with any code or other content which is covered by a license that would directly or indirectly require that all or part of the Licensed Technology be governed under any terms other than those of this Agreement (“Non-Compatible License”). Code or content under the following licenses, for example, are prohibited: GNU General Public License (GPL), Lesser GPL (LGPL) (unless you are merely dynamically linking a shared library), or Creative Commons Attribution-ShareAlike License. Code or content under the following licenses, for example, are allowed: BSD License, MIT License, Microsoft Public License, or Apache License. You may not sublicense the Licensed Technology under a Non-Compatible License." - [Unreal Engine EULA](https://www.unrealengine.com/en-US/eula/publishing)

Please change the license so I can use this. :)

**Comment by Jordonbc** [comment]
Hello @HeadClot, Thanks for using my plugin. I have changed the license to Apache 2.0.
